lib/jwt/jwt.go

```diff
@@ -10,5 +10,5 @@ type Signer interface {
 }
 
 type Verifier interface {
-	Verify(ctx context.Context, token string) (claims Claims, err error)
+	Verify(ctx context.Context, token string, allowedType string, allowedVersions []string) (claims Claims, err error)
```

Comment:
might work well as type+version pairs

Reply:
Is there a case for moving the prefixing etc to this layer too? I suspect we will never use JWTs without that